backend/medtagger/database/models.py

```diff
@@ -249,31 +243,54 @@ def __init__(self, user: User, labeling_time: LabelingTime) -> None:
         By default all of the labels are not verified
         """
         self.id = LabelID(str(uuid.uuid4()))
-        self.status = LabelStatus.NOT_VERIFIED
         self.owner = user
         self.labeling_time = labeling_time
 
     def __repr__(self) -> str:
         """Return string representation for Label."""
-        return '<{}: {}: {} {} {} {}>'.format(self.__class__.__name__, self.id, self.scan_id, self.status,
-                                              self.labeling_time, self.owner)
+        return '<{}: {}: {} {} {}>'.format(self.__class__.__name__, self.id, self.scan_id,
+                                           self.labeling_time, self.owner)
+
 
-    def update_status(self, status: LabelStatus) -> 'Label':
-        """Update Label's status.
+class LabelTag(Base):
+    """Definition of tag for label."""
 
-        :param status: new status for this Label
-        :return: Label object
+    __tablename__ = 'LabelTags'
+    id: LabelTagID = Column(String, autoincrement=True, primary_key=True)
+    key: str = Column(String(50), nullable=False, unique=True)
+    name: str = Column(String(100), nullable=False)
+
+    label_elements: 'LabelElement' = relationship("LabelElement", back_populates="tag")
```

Comment:
I'm not sure if this is necessary. Why would you like to get all LabelElements for a tag?

Example:
```
tag = LabelTag.query().filter_by(key='LEFT_KIDNEY').one()
print(tag.label_elements)  # why?
```